spec/facades/route_facade_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe 'Route Facade' do
+  it 'collects route data' do
+    to = "1225 17th Street, Suite 130, Denver, CO 80202"
+    from = "1331 17th St, Ste LL100, Denver, CO 80202"
+    route = RouteFacade.directions(to, from)
+
+    expect(route).to be_a Route
```

Comment:
I'd recommend adding more assertions here about the Route object's attributes (and the same feedback would apply to the Station Facade test).